R/methods.R

```diff
@@ -1,17 +1,85 @@
-#' Title
+#' Imputation & analysis Methods
+#'
+#' @description
+#'
+#' These functions determine what methods rbmi should use when creating
+#' the imputation models, generating imputed values and  pooling the results.
 #'
 #' @name method
-#' @param burn_in TODO
-#' @param burn_between TODO
-#' @param same_cov TODO
-#' @param n_samples TODO
-#' @param covariance TODO
-#' @param threshold TODO
-#' @param REML TODO
-#' @param type TODO
-#' @param verbose TODO
-#' @param seed TODO
-#' @return TODO
+#'
+#' @param burn_in a numeric that specifies how many observations should be discarded
+#' prior to extracting actual samples. Note that the sampler
+#' is initialised at the maximum likelihood estimates and an uninformative
+#' prior is used thus in theory this value should not need to be that high.
```

Comment:
Should we specify the default values?

Reply:
I'm going to go for  no....  only because in theory they can vary between function and also the usage function should show what they are e.g.:

![image](https://user-images.githubusercontent.com/8447209/133256101-ba2e060d-096c-4413-bfcd-e4980fd71c55.png)